### PR TITLE
[Crate] Bump rustfmt version to 1.6.0

### PR DESCRIPTION
**Pull request by Douile**

### CODE REVIEW

File: src/errors.rs
[the remark this answers is not in the document]
Comment (Douile) [reply src/errors.rs:1]: It's the line endings, changing from CRLF to LF (cargo fmt did this).

[the remark this answers is not in the document]
Comment (CosminPerRam) [reply src/errors.rs:1]: Huh? Aren't all files uploaded to github by default ending with LF? I work from both Windows and Linux and when i `git add` git tells me that all CRLF instances will be changed to LF on commit.

[the remark this answers is not in the document]
Comment (Douile) [reply src/errors.rs:1]: You can configure it to turn it off. On main branch this file appears to be CRLF.

[the remark this answers is not in the document]
Comment (Douile) [reply src/errors.rs:1]: In the initial commit: https://github.com/gamedig/rust-gamedig/commit/8098136d0983c72b4d0c0545b59d28fd3828b4be all the files have CRLF, but src/games/mod.rs appears to have changed at some point. I don't know why this one hasn't maybe just hasn't been touched as much.

**Comment by CosminPerRam** [comment]
I checked with both IntelliJ CLion and Notepad (Windows) and: 
- on this branch `src/errors.rs` has `CRLF`, on main it has `CRLF`.
- on this branch `src/socket.rs` has `CRLF`, on main it has `CRLF`.

So I still don't understand whats changed.

File: src/errors.rs
[the remark this answers is not in the document]
Comment (CosminPerRam) [reply src/errors.rs:1]: I checked with both IntelliJ CLion and Notepad (Windows) and:

- on this branch src/errors.rs has CRLF, on main it has CRLF.
- on this branch src/socket.rs has CRLF, on main it has CRLF.

So I still don't understand whats changed.

**Pull request merged**